spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before :each do
+    @user = User.create(name: 'olla', photo: 'https://picsum.photos/id/23/200', bio: 'web developer',
+                        posts_counter: 0)
+    visit root_path
+  end
+
+  it 'shows the correct content' do
+    expect(page).to have_content('Welcome To User Page')
+  end
```

Comment:
- [ ] Please create more users in the `before` block so that you can then write tests that assert that you can see the username of all the other others